src/rules/javascript/webextension-unsupported-api.js

```diff
@@ -0,0 +1,21 @@
+import { UNSUPPORTED_API } from 'messages/javascript';
+import { hasBrowserApi } from 'schema/browser-apis';
+
+export default {
+  create(context) {
+    return {
+      MemberExpression: function(node) {
+        if (node.object.object &&
```

Comment:
Looking at some eslint examples it seems like `node.object` is always available (at least in a `MemberExpression`) so this should be fine.